docs/tools/fbsql/fbsql-loaders-kafka.md

```diff
@@ -1,71 +1,41 @@
 ---
-title: fbsql-Kafka
+title: fbsql-loader-kafka
 layout: default
 parent: fbsql
 grand_parent: Tools
 nav_order: 2
 ---
 
```

Comment:
Page title doesn't give context on what this is going to do and in what context. An alternative is something like:

"How do I load Kafka data using fbsql?"


Also needs:
* An introduction to explain what this is (FBSQL Kafka loader does x, y, z)
* Dependencies and requirements to perform under "Before you begin" heading because the user can't just leap in and do this can they?